tests/postman/newman/collection.json

```diff
@@ -13576,6 +13576,267 @@
+                  "    var jsonData = pm.response.json();",
+                  "    pm.response.to.have.status(200);",
+                  "    pm.environment.set(\"customer-id\", jsonData.customer_id);",
+                  "});"
+                ]
+              }
+            },
+            {
+              "listen": "prerequest",
+              "script": {
+                "id": "f4a1b51e-3eea-4ae6-9dfd-f4c5faf33af3",
+                "type": "text/javascript",
+                "exec": [
+                  "pm.environment.unset(\"customer_id\");"
+                ]
+              }
+            }
+          ],
+          "request": {
+            "auth": {
+              "type": "bearer",
+              "bearer": [
+                {
+                  "key": "token",
+                  "value": "{{customer-access-token}}",
+                  "type": "string"
+                }
+              ]
+            },
+            "method": "GET",
+            "header": [
+              {
+                "key": "Content-Type",
+                "value": "application/json"
+              }
+            ],
+            "body": {
+              "mode": "raw",
+              "raw": ""
+            },
+            "url": {
+              "raw": "{{domain}}{{endpoint_customers_me}}",
+              "host": [
+                "{{domain}}{{endpoint_customers_me}}"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "Update customer data",
+          "event": [
+            {
+              "listen": "test",
+              "script": {
+                "id": "898b977f-9f9e-40b8-8201-4352c18c619b",
+                "type": "text/javascript",
+                "exec": [
+                  "pm.test(\"Status code is 200\", function () {",
+                  "    pm.response.to.have.status(200);",
+                  "});"
+                ]
+              }
+            },
+            {
+              "listen": "prerequest",
+              "script": {
+                "id": "878ccafd-8752-4982-8896-5f83b8e7aebb",
+                "type": "text/javascript",
+                "exec": [
+                  ""
+                ]
+              }
+            }
+          ],
+          "request": {
+            "auth": {
+              "type": "bearer",
+              "bearer": [
+                {
+                  "key": "token",
+                  "value": "{{customer-access-token}}",
+                  "type": "string"
+                }
+              ]
+            },
+            "method": "POST",
+            "header": [
+              {
+                "key": "Content-Type",
+                "value": "application/json"
+              }
+            ],
+            "body": {
+              "mode": "raw",
+              "raw": "{\n    \"email\": \"should@not.change\",\n    \"firstname\": \"newFirstName\",\n    \"middlename\": \"newMiddleName\",\n    \"lastname\": \"newLastName\"\n}"
```

Comment:
DOB and gender are also part of the default set, yea? Double check the mage 1800 instance if it has those modifiable.